app.js

```diff
@@ -0,0 +1,18 @@
+import { giftData } from './api.js';
+
+const randomNumber1 = Math.floor(Math.random() * 20 + 1);
```

Comment:
the +1 will never give you index of 0 and occasionally will give you an index of 21, which does not exist